cmd/loadgenerator/config.go

```diff
@@ -0,0 +1,111 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
There is a decision to make one whether each input should be it's own binary, or if we should start with a single binary that starts a specific input based on the command it is given e.g. `agent-inputs loadgenerator` or similar.

I think I would favour only having a single binary initially to simplify the build and distribution and we can split it up later if it gets too large or difficult to work with.

Reply:
Makes sense. I'll refactor it.